chrono/src/chrono.cpp

```diff
@@ -53,6 +53,10 @@ chrono_t::notify_every(double time, bool send_id) {
 cocaine::streamed<io::timer_id_t>
 chrono_t::set_timer_impl(double first, double repeat, bool send_id) {
     streamed<io::timer_id_t> promise;
+    if(repeat != 0.0 && repeat * 1000000.0 < minimal_interval_us) {
```

Comment:
Comparing with `0.0` is not specified. Comparing with zero duration is specified - use it.